airbyte-integrations/connectors/source-redshift/src/main/java/io/airbyte/integrations/source/redshift/RedshiftSource.java

```diff
@@ -51,6 +69,25 @@ private void addSsl(final List<String> additionalProperties) {
     additionalProperties.add("sslfactory=com.amazon.redshift.ssl.NonValidatingFactory");
   }
 
+  @Override
+  public List<TableInfo<CommonField<JDBCType>>> discoverInternal(JdbcDatabase database) throws Exception {
```

Comment:
is this still necessary? Looks like we're also setting the `currentSchema` additionalProperty, is that not sufficient?

Reply:
Yup, jdbc returns all schemas. If we do not filter it explicitly - then all schemas would be shown on UI on the discovery stage similarly to Postgres and oracle connectors

Reply:
got it. what does the `currentSchema` property do then?

Reply:
The `currentSchema` property will limit the query to the sepcified schemas.